avaframe/com1DFA/com1DFA.py

```diff
@@ -327,16 +346,21 @@ def reportAddTimeMassInfo(reportDict, tcpuDFA, infoDict):
     """ Add time and mass info to report """
 
     # add mass info
-    reportDict['Simulation Parameters'].update({'Initial mass [kg]': ('%.2f' % infoDict['initial mass'])})
-    reportDict['Simulation Parameters'].update({'Final mass [kg]': ('%.2f' % infoDict['final mass'])})
-    reportDict['Simulation Parameters'].update({'Entrained mass [kg]': ('%.2f' % infoDict['entrained mass'])})
-    reportDict['Simulation Parameters'].update({'Entrained volume [m3]': ('%.2f' % infoDict['entrained volume'])})
+    reportDict['Simulation Parameters'].update(
+        {'Initial mass [kg]': ('%.2f' % infoDict['initial mass'])})
+    reportDict['Simulation Parameters'].update(
+        {'Final mass [kg]': ('%.2f' % infoDict['final mass'])})
```

Comment:
There's now a ton of line breaks which are way below our 120 characters. Could it be that your editor is set to 80 character line breaks? (i am not commenting on all these lines...)